code/modules/admin/verbs/adminhelp.dm

```diff
@@ -513,8 +539,10 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		else
 			current_ticket.AddInteraction("[key_name_admin(usr)] opened a new ticket.")
 			current_ticket.Close()
-
-	new /datum/admin_help(msg, src, FALSE)
+	if(alert(usr,"Is this an administrative issue, or a gameplay question?",,"Admin issue","Question") == "Question")
+		new /datum/admin_help(msg, src, FALSE, TRUE)
+	else
+		new /datum/admin_help(msg, src, FALSE, FALSE)

```

Comment:
.

Reply:
These should just be different verbs anyways. No reason to combine them.

Reply:
I'm really just here to tell people how to optimize their code not how to design their code.At the end of the day,it just depends on how much we want to trust mentors with admin info by giving them an admin datum(giving access to read-only VV and tons of round info) or making a new system of mentor datums just for them.